File: ES6/js/triangle.js
/*
export let width=100;
export let height=100;
export const 면적=(width,height)=>width*height*0.5;
*/


let width=100;
let height=100;
const 면적=(width,height)=>width*height*0.5;

console.log('삼각형 가로폭:',width);
console.log('삼각형 세로폭:',height);
console.log('면적:',면적(100,100));
console.log('{width,height,면적}출력:',{width,height,면적});
//export {키:값,....} 시 문법 오류 즉 shorthand property 기법을 적용해야 한다
//또한 export default로 export시는  {width,height,면적}로 받을 수 없다
export {width,height,면적};

File: Java/Basic/javaSEProj/src/interface18/Triangle.java
package interface18;

public class Triangle extends FigureData implements Figure,Drawble{//클래스1개, 인터페이스2개 상속

	public Triangle(int width, int height) { //가로폭, 세로폭 초기화
		super(width, height);
	}
	@Override
	public void draw(String figureName) {
		System.out.println(figureName+"을 3개의 선으로 그리다");
	}
	@Override
	public void area(String figureName) {
		System.out.println(figureName+"의 면적:"+(int)(width*height*0.5));
	}

}
